src/ui/Views/Course/Variants.cshtml

```diff
@@ -20,32 +20,29 @@
 </div>
 
 <ul>
-    @foreach (var course in Model.Courses)
-    {
-
     <h1 class="heading-xlarge">
-        <span class="heading-secondary">@course.Name</span>
-        PGCE with QTS full time (@course.ProgrammeCode)
+        <span class="heading-secondary">@Model.Course.Name</span>
+        PGCE with QTS full time (@Model.Course.ProgrammeCode)
     </h1>
         <dl class="aligned">
             <dt>Name:</dt>
-            <dd>@course.Name</dd>
+            <dd>@Model.Course.Name</dd>
             <dt>Training provider code:</dt>
-            <dd>@course.ProviderCode</dd>
+            <dd>@Model.Course.ProviderCode</dd>
             <dt>Training programme code:</dt>
-            <dd>@course.ProgrammeCode</dd>
+            <dd>@Model.Course.ProgrammeCode</dd>
             <dt>Accredited provider:</dt>
```

Comment:
This was a bug in the prototype, can it be "Accrediting provider:"

Reply:
ok